cmd/crane/cmd/mutate.go

```diff
@@ -153,7 +160,8 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 	mutateCmd.Flags().StringToStringVarP(&envVars, "env", "e", nil, "New envvar to add")
 	mutateCmd.Flags().StringSliceVar(&entrypoint, "entrypoint", nil, "New entrypoint to set")
 	mutateCmd.Flags().StringSliceVar(&cmd, "cmd", nil, "New cmd to set")
-	mutateCmd.Flags().StringVarP(&newRef, "tag", "t", "", "New tag to apply to mutated image. If not provided, push by digest to the original image repository.")
+	mutateCmd.Flags().StringVarP(&newRepo, "repo", "r", "", "Repository to push the mutated image to. If provided, push by digest to this repository.")
```

Comment:
```suggestion
	mutateCmd.Flags().StringVar(&newRepo, "repo", "", "Repository to push the mutated image to. If provided, push by digest to this repository.")
```

I don't think `--repo` is a common enough use case to warrant the shorthand, and it may be nice to keep available for later.